config/routes.rb

```diff
@@ -3,4 +3,9 @@
 
   # Defines the root path route ("/")
   # root "articles#index"
+  resources :flights, only: [:index] do
+    resources :flight_passengers, only: [:destroy]
+  end
```

Comment:
Although you are making use of advanced routing here, what you have here is incorrect. `FlightPassengers` is not a child of `Flights`. The path is incorrect.